cmd/server/http/http.go

```diff
@@ -265,12 +266,12 @@ func daemonWebhook(flowSvc *flow.Service) http.HandlerFunc {
 
 		err = flowSvc.NotifyCommitter(context.Background(), &podNotify)
 		if err != nil {
-			logger.Errorf("http: daemon webhook failed: pod '%s' namespace '%s': notify committer: %v", podNotify.Name, podNotify.Namespace, err)
+			logger.Errorf("http: daemon webhook failed: pod '%s' namespace '%s' environment: %s notify committer: %v", podNotify.Name, podNotify.Namespace, podNotify.Environment, err)
```

Comment:
Can we add `'` quotes around the environment string like you do below? This will make it super clear if the field for some reason is empty.